client/my-sites/people/viewers-list/invite-button.jsx

```diff
@@ -0,0 +1,13 @@
+import { Button, Gridicon } from '@automattic/components';
+import { localize } from 'i18n-calypso';
```

Comment:
Since this is a functional component, I'd suggest using `useTranslate()` hook instead of the `localize()` HoC.